frontend/packages/topology/src/components/graph-view/components/withTopologyContextMenu.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {
+  GraphElement as TopologyElement,
```

Comment:
nit: There is no name conflict with another GraphElement in this file, so we can just use GraphElement here, or?